src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/system_information.html.twig

```diff
@@ -117,7 +117,7 @@
             </div>
           </div>
         </div>
-    </div>
+      </div>
```

Comment:
I just noticed that this div closing tag should be outside the %if statement, can you confirm?

Reply:
![image](https://user-images.githubusercontent.com/793712/95565672-71393700-0a18-11eb-98e2-7b579c952082.png)

Reply:
Maybe it's time to change the whole indentation of this file and use 2spaces? :D

Reply:
Will gladly do it, I will rebase and push